test/envoye2e/stackdriver_plugin/stackdriver.go

```diff
@@ -120,11 +128,16 @@ func (sd *Stackdriver) Cleanup() {
 
 func (sd *Stackdriver) Check(p *driver.Params, tsFiles []string, lsFiles []SDLogEntry, verifyLatency bool) driver.Step {
 	// check as sets of strings by marshaling to proto
-	twant := make(map[string]struct{})
+	twant := make(map[string]int64)
 	for _, t := range tsFiles {
 		pb := &monitoring.TimeSeries{}
 		p.LoadTestProto(t, pb)
-		twant[prototext.Format(pb)] = struct{}{}
+		if len(pb.Points) != 1 || pb.Points[0].Value.GetInt64Value() == 0 {
+			panic("malformed metric golden")
```

Comment:
should we have a `ValidateGolden` pre-step that can fail (t.Fatalf) tests without panics as a better option here?

Reply:
Seems like too much work for just one extension. I think panics will be caught early anyways.

Reply:
could we `log.Fatalf` here instead?  i know that doesn't make a ton of difference in the end, but I have a strong aversion to seeing `panic` in code (for better or worse).

Reply:
Done.